src/libraries/System.Runtime.Extensions/tests/System/Random.cs

```diff
@@ -753,7 +753,41 @@ public static void Shuffle_Span_Seeded(bool emptyShuffle)
             }
 
             random.Shuffle(items);
-            AssertExtensions.SequenceEqual(stackalloc int[] { 1, 4, 3, 2 }, items);
+            AssertExtensions.SequenceEqual(stackalloc int[] { 2, 1, 4, 3 }, items);
+        }
+
+        [Fact]
+        public static void Shuffle_Array_Fairness()
+        {
+            // Test that repeatedly shuffling an array puts each value in each position a roughly equal number of times.
+            // This does not prove that the shuffle is fair but will at least fail if the shuffle is obviously unfair
+            const int runs = 10000;
+            const int len = 50;
```

Comment:
would it be worth changing to a `[Theory]` so you can do this for several values of `len` - maybe at least an odd number as well as an even number or something?